proto/src/determined/api/v1/api.proto

```diff
@@ -2794,4 +2795,71 @@ service Determined {
       tags: "Internal"
     };
   }
+
+  // Add or update workspace task config policies.
+  rpc PutWorkspaceConfigPolicies(PutWorkspaceConfigPoliciesRequest)
+      returns (PutWorkspaceConfigPoliciesResponse) {
+    option (google.api.http) = {
+      put: "/api/v1/config-policies/workspaces/{workspace_id}/{workload_type}"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"  // FIXME what tag to use?
```

Comment:
Do we want to use a tag like "alpha" or "not implemented yet" so customers don't try to use it while digging through swagger?

Reply:
That's a cool idea! I do think "Internal" should be a clear enough indication that it's not something they should attempt to interact with, but it wouldn't surprise me if somebody has already implemented things that interact with our Internal API endpoints. So I like the idea of having something like an Alpha section a lot!